### PR TITLE
Update reacji row styles.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

cc @buoyad

### CODE REVIEW

File: shared/chat/conversation/messages/wrapper/index.js [hunk 1]
```diff
@@ -416,10 +416,7 @@ class _WrapperMessage extends React.Component<Props & Kb.OverlayParentProps, Sta
                     conversationIDKey={this.props.conversationIDKey}
                     onShowingEmojiPicker={this._setShowingPicker}
                     ordinal={message.ordinal}
-                    style={Styles.collapseStyles([
-                      styles.emojiRow,
-                      this.props.isLastInThread ? styles.emojiRowLast : null,
```
Comment (buoyad) [inline shared/chat/conversation/messages/wrapper/index.js]: Now this prop `isLastInThread` is unused, can we remove it?

Comment (buoyad) [reply shared/chat/conversation/messages/wrapper/index.js]: Whoops too late

Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/index.js]: Sure! I didn't look to see if it was used elsewhere. I'll open another PR removing it.

Comment (buoyad) [reply shared/chat/conversation/messages/wrapper/index.js]: Thanks!

**Pull request merged**